traffic_router/core/src/main/java/com/comcast/cdn/traffic_control/traffic_router/core/dns/ZoneManager.java

```diff
@@ -458,8 +458,8 @@ private static void generateZones(final TrafficRouter tr, final LoadingCache<Zon
 
 	@SuppressWarnings({"PMD.ExcessiveParameterList"})
 	private static List<Record> createZone(final String domain, final Map<String, List<Record>> zoneMap, final Map<String, DeliveryService> dsMap,
-			final TrafficRouter tr, final LoadingCache<ZoneKey, Zone> zc, final LoadingCache<ZoneKey, Zone> dzc, final List<Runnable> generationTasks,
```

Comment:
Can I trouble you to undo the extraneous formatting changes in these changed files?